src/words/TokTok.jl

```diff
@@ -44,21 +44,15 @@ const CLOSE_PUNCT =(
         "\u301e", "\u301f", "\ufd3f", "\ufe18", "\ufe36", "\ufe38", "\ufe3a", "\ufe3c",
         "\ufe3e", "\ufe40", "\ufe42", "\ufe44", "\ufe48", "\ufe5a", "\ufe5c", "\ufe5e",
         "\uff09", "\uff3d", "\uff5d", "\uff60")
-    
-# This is the \p{Close_Punctuation} from Perl's perluniprops             
+
+# This is the \p{Close_Punctuation} from Perl's perluniprops
 const CURRENCY_SYM = (
-        "\u00a2", "\u00a3", "\u00a4", "\u00a5", "\u058f", "\u060b", "\u09f2", "\u09f3", 
+        "\u00a2", "\u00a3", "\u00a4", "\u00a5", "\u058f", "\u060b", "\u09f2", "\u09f3",
         "\u0af1", "\u0bf9", "\u0e3f", "\u17db", "\u20a0", "\u20a1", "\u20a2", "\u20a3",
         "\u20a4", "\u20a5", "\u20a6", "\u20a7", "\u20a8", "\u20a9", "\u20aa", "\u20ab",
         "\u20ac", "\u20ad", "\u20ae", "\u20af", "\u20b0", "\u20b1", "\u20b2", "\u20b3",
         "\u20b4", "\u20b5", "\u20b6", "\u20b7", "\u20b8", "\u20b9", "\u20ba", "\ua838",
-        "\ufdfc", "\ufe69", "\uff04", "\uffe0", "\uffe1", "\uffe5", "\uffe6", "\u09fb",)
-
-# Left/Right strip, i.e. remove heading/trailing spaces.
-const LSTRIP = (" ",) => ""           
-const RSTRIP = ("\r", "\n", "\t", "\f", "\v",) => "\n"  
```

Comment:
These rules were declared but wasn't used original toktok perl script. Thus, removed this.